turbopack/crates/turbo-tasks-backend/src/backend/storage.rs

```diff
@@ -205,7 +205,7 @@ pub struct InnerStorage {
     aggregation_number: OptionStorage<AggregationNumber>,
     output_dependent: AutoMapStorage<TaskId, ()>,
     output: OptionStorage<OutputValue>,
-    upper: AutoMapStorage<TaskId, i32>,
+    upper: AutoMapStorage<TaskId, u32>,
```

Comment:
just curious: could this be `NonZeroU32`?  i doubt there are any representational benefits, but it might be useful as documentation

Reply:
I think it could be... I'll make a follow-up PR